### ISSUE TITLE
pivot error

### ISSUE THREAD

**Issue by ericemc3** [uwdata/arquero#133]
I just found an observable notebook 'broken', and i am wondering if this about Arquero last version:

Here is a reprex:
```
dt = aq.table({
    country: ["France", "France", "France", "Germany", "Germany", "Germany", "Japan", "Japan", "Japan"],
    year: [2017, 2018, 2019, 2017, 2018, 2019, 2017, 2018, 2019], 
    expenditure: ['NA', 51410, 52229, 45340, 46512, 51190, 46542, 46618, 46562]})
```
step2:
```
dt2 = dt.filter(d => d.year > 2017)
        .orderby('country')
```

next block yields an error: _TypeError: Cannot read property 'count' of undefined_
```
dt2.groupby('country')
   .pivot('year', 'expenditure')
```
The error seems to result from the combination, in step2, of `filter` and `orderby`.

I have temporariliy inserted a dt2.reify() before final groupby/pivot

**Comment by jheer** [comment]
Thanks for sharing, I’ll take a look soon!

**Comment by jheer** [comment]
Fixed released in v4.1.2, and updated on Observable.

**Comment by ericemc3** [comment]
Thnaks a lot, it works in Observable again!
